pulpcore/cli/common/debug.py

```diff
@@ -43,3 +51,48 @@ def task_summary(pulp_ctx: PulpContext) -> None:
         answer = pulp_ctx.call(PulpTaskContext.LIST_ID, parameters=payload)
         result[state] = answer["count"]
     pulp_ctx.output_result(result)
+
+
+@debug.group(name="openapi")
+def openapi_group() -> None:
+    pass
+
+
+@openapi_group.command()
+@click.option("--jq", "jq_expr")
+@pass_pulp_context
+def schema(pulp_ctx: PulpContext, jq_expr: Optional[str]) -> None:
+    """
+    Print the openapi schema of the server.
+    """
+    result = pulp_ctx.api.api_spec
+    if jq_expr:
+        if jq is None:
+            raise click.ClickException(_("`jq` is not installed."))
+        else:
+            result = jq.compile(jq_expr).input(result).first()
```

Comment:
I just now realized, we could make this a global option to transform any old (json/yaml based) output. WDYT?

Reply:
I like this idea. I'd divorce it from this PR - open an issue and expand on this once it's merged maybe?